src/components/FormattedText/FormattedText.test.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import FormattedText from "./FormattedText";
+
+it("should render a text with a default font-size of 16px", () => {
+  const { getByText } = render(<FormattedText>My text</FormattedText>);
+
+  const mainText = getByText("My text");
+  expect(mainText).toHaveStyle({ fontSize: "16px" });
+});
+
+test.each([
+  ["-2", "12"],
```

Comment:
Can you also add a test case for the default case?
```suggestion
  [undefined, "16"],
  ["-2", "12"],
```

Reply:
There is already a separate test for the default case. 
If I add `[undefined, "16"]` to that array I get type error because the `size` property in Props doesn't contain the undefined case `size: '-2' | '-1' | '0' | '1' | '2' | '3' | '4' | '5' | '6' | '7' | '8';`. If I add the `undefined`case there I get another error because `undefined` can't be used as an index so I would need to change the `fontSizes` object to a method I think or add something like this in the CSS `font-size: ${props.size === undefined ? '16px' : fontSizes[props.`. 

Should I do this or is the separate test case ok?

Reply:
Then nevermind